README.md

```diff
@@ -6,6 +6,8 @@
   <a href="https://github.com/actions/setup-ruby/actions"><img alt="versions status" src="https://github.com/actions/setup-ruby/workflows/ruby-versions/badge.svg"></a>  
 </p>
 
+**Please note: This repository is currently unmaintained by a team of developers at GitHub, we are not going to be updating issues or pull requests on this repository. Please, migrate your workflows to [ruby/setup-ruby](https://github.com/ruby/setup-ruby) by official [Ruby community](https://www.ruby-lang.org/en/).**
```

Comment:
I hope this doesn't come off as rude, but this sounds a bit off to me. I don't know if it's a case of a typo or just regional differences in English, but the following sounds better to me:

> Please note: This action is deprecated and should no longer be used. The team at GitHub has ceased making code contributions to the project and will not be maintaining the issue tracker or accepting pull requests. Please migrate your workflows to the the [ruby/setup-ruby](https://github.com/ruby/setup-ruby) action, which is being actively maintained by members of the Ruby community.